package.json

```diff
@@ -65,6 +65,7 @@
         "jwt-decode": "^2.2.0",
         "prop-types": "^15.7.2",
         "react": "^16.8.6",
+        "react-autosuggest": "^9.4.3",
```

Comment:
Hello @CryceTruly what does this package do??

Reply:
Its a react library that eases displaying a list of suggestions.